torch/_dynamo/comptime.py

```diff
@@ -297,6 +301,14 @@ def print_guards():
     comptime(lambda ctx: ctx.print_guards())
 
 
+def breakpoint():
+    def inner(inner_ctx):
+        ctx = inner_ctx.parent()
+        builtins.breakpoint()
+
+    comptime(inner)
```

Comment:
Noob question, but how do you actually use this API? @comptime.breakpoint on a function?

(nit: could use a quick example in a docstring, so other ppl can figure out how to use it)

Reply:
Just comptime.breakpoint(), just like you do a normal breakpoint